xml/admin_ceph_upgrade.xml

```diff
@@ -245,10 +245,22 @@
     &prevproductnumber; are applied to all cluster nodes:
    </para>
 <screen>&prompt.root;zypper refresh &amp;&amp; zypper patch</screen>
+   <tip>
+    <para>
+     Refer to <link
+     xlink:href="https://documentation.suse.com/ses/6/html/ses-all/storage-salt-cluster.html#deepsea-rolling-updates"/>
+     for detailed information about updating the cluster nodes.
+    </para>
+   </tip>
    <para>
-    After updates are applied, check the cluster health:
+    After updates are applied, restart the &smaster;, synchronize new &salt;
+    modules, and check the cluster health:
    </para>
-<screen>&prompt.cephuser;ceph -s</screen>
+<screen>
+&prompt.smaster;systemctl restart salt-master.service
+&prompt.smaster;salt "*" saltutil.sync_all
```

Comment:
One tiny thing: I think we've used single quotes elsewhere (e.g. `salt '*'` vs `salt "*"`)

Reply:
you are right, copy'n'pasta mistake